### PR TITLE
Remove FQDN in favour of inventory_hostname

### PR DESCRIPTION
**Pull request by oscarkraemer**
fqdn might be confused with ansible_fqdn which requires gather_facts which might be different on different nodes. By using ansible_inventory we can be sure that the fact is always correct.

This should be reviewed as part of similar PRs.
CCCP-2888

### CODE REVIEW

**Comment by khabiruddin** [comment]
Looks good to me.

File: tests/test.yml [hunk 1]
```diff
@@ -40,7 +40,6 @@
              address: "{{ internal_ip }}"
              netmask: 255.255.255.0
        - bridges: [ "br-example" ]
-       - fqdn: "localhost"
```
Comment (Aegil) [inline tests/test.yml]: should this be replaced with something?

Comment (oscarkraemer) [reply tests/test.yml]: No, IMHO. It should really on the inventory to work. If I should not remove that variable the test might not be correctly tested.

**Comment by Aegil** [comment]
looks good

**Pull request merged**